homeassistant/components/ozw/discovery.py

```diff
@@ -131,6 +131,37 @@
             },
         },
     },
+    {  # Z-Wave Thermostat device without mode support
+        const.DISC_COMPONENT: "climate",
+        const.DISC_GENERIC_DEVICE_CLASS: (const_ozw.GENERIC_TYPE_THERMOSTAT,),
+        const.DISC_SPECIFIC_DEVICE_CLASS: (
+            const_ozw.SPECIFIC_TYPE_SETPOINT_THERMOSTAT,
+        ),
+        const.DISC_VALUES: {
+            const.DISC_PRIMARY: {
+                const.DISC_COMMAND_CLASS: (CommandClass.THERMOSTAT_SETPOINT,)
+            },
+            "temperature": {
+                const.DISC_COMMAND_CLASS: (CommandClass.SENSOR_MULTILEVEL,),
+                const.DISC_INDEX: (1,),
```

Comment:
The magic numbers should probably be enum constants in the library.

Reply:
Yes, we need to address constants for the ValueIndexes but that needs some discussion as ValueIndex can be the same between CC's. So if we want to do it 100% right, we should create the ValueIndex Enum/Constants for each CommandClass.

There's already an issue open for that in the lib as a reminder ;-)